packages/tabs/src/Tabs.tsx

```diff
@@ -99,7 +99,9 @@ function Tabs({
   as = 'button',
   ...rest
 }: TabsProps) {
-  const childrenArray = React.Children.toArray(children);
+  const childrenArray = React.Children.toArray(children) as Array<
+    React.ReactElement
+  >;
```

Comment:
I don't know why this wasn't caught before 🤔

Reply:
Does toArray return `Array<React.ReactNode>`?

As an aside, it looks like this component doesn't have any specific code preventing a non-Tab ReactElement from being passed in, but there's some assumptions made that it will always be a tab (or have the same props as a Tab) when rendering. Might be worth looking into a bit more at a later time.